src/main/resources/application.yaml

```diff
@@ -100,7 +100,7 @@ exui.role:
 prdEnumRoleType: ADMIN_ROLE,JURISD_ID
 jurisdictionIdType: ADMIN_ROLE,CCD_ROLE,SIDAM_ROLE
 
-userProfUrl: ${USER_PROFILE_URL:https://rd-user-profile-api-pr-108.service.core-compute-preview.internal}
+userProfUrl: ${USER_PROFILE_URL:https://rd-user-profile-api-pr-115.service.core-compute-preview.internal}
```

Comment:
Change to api preview

Reply:
For this separate JIRA has raised and will be fixed under that.